packages/components/src/local/organisms/planning-channel/index.tsx

```diff
@@ -366,23 +370,16 @@ export const PlanningChannel: React.FC<PropTypes> = ({
     }
   }, [viewAsForce])
 
-  useEffect(() => {
-    // only update selected assets if we're not planning an activity
-    if (!activityBeingPlanned) {
-      setSelectedAssets(localSelectedAssets)
-    }
-  }, [localSelectedAssets])
-
   useEffect(() => {
     // produce the own and opp assets for this player force
     const forceCols = getForceColors(allForces)
     const platIcons = platformIcons(platformTypes)
```

Comment:
hello @lebaphi -  your code has dropped the extra parameter for the next two calls - that will prevent some functionality from working.